docs/changelog.rst

```diff
@@ -15,6 +15,8 @@ New flavors:
 
 New fields for existing flavors:
 
+- Added ARCBUField model field.
```

Comment:
@benkonrath Damn, missed that one. It's fixed now, I hope I didn't miss anything else. Thanks again, I'll be making another PR on the following days, please stay around! ;P